config/services/s3.toml

```diff
@@ -2,9 +2,14 @@
 # Note that this internally uses opendal S3 service, see https://opendal.apache.org/docs/rust/opendal/services/struct.S3.html
 # where endpoint, bucket and root are extracted from the repository URL.
 [repository]
-repository = "s3:https://s3.amazonaws.com/bucket/my/repopath"
-password = "mypassword"
+repository = "opendal:s3"
+password = "password"
 
 # Other options can be given here - note that opendal also support reading config from env files or AWS config dirs, see the opendal S3 docu
 [repository.options]
-region = "us-east-1"
+access_key_id = "xxx"
+secret_access_key = "xxx"
+bucket = "bucket_name"
+endpoint = "https://s3.amazonaws.com"
+region = "auto" # Explicit region is better, else requests are delayed to determine correct region.
```

Comment:
done

Reply:
I wonder, what about other products, like: https://docs.aws.amazon.com/general/latest/gr/glacier-service.html | `glacier.us-east-1.amazonaws.com` maybe that works, too?

Reply:
I think so, see `default_storage_class`. But I haven't tested it. Also, this should go into a separate example config as this will be a hot/cold repo, then.